pages/morse.js

```diff
@@ -76,7 +77,7 @@ export default function ButtonAppBar() {
           sx={{ color: 'primary.main' }}
         >
           <Box className={layout_styles.inputs_box}>
```

Comment:
Shrink inputs_box as much as possible, e.g. by
- removing flex-basis on it
- increasing flex-grow on results_box